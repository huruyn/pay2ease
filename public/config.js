window.API_URL = "https://zippy-retail-api-dev.azurewebsites.net";