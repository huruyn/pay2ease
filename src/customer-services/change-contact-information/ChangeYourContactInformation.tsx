import {
  Col,
  Row,
  Card,
  Form,
  Button,
} from 'react-bootstrap';
import CommonPageContainer from '../../common/CommonPageContainer';

export default function ChangeYourContactInformation() {
  return (
    <div className="page-content">
      <CommonPageContainer title="Change Your Contact Information">
        <Row>
          <Col md={8}>
            <Row className="mt-3">
              <Col>
                <b>Step: 1 - 2</b>
              </Col>
            </Row>
            <h4 className="mt-4">Enter and verify your contact information</h4>
            <p className="mt-3">
              ZippyX will use this new email address to communicate with you for
              many reasons including sending you promotional materials,
              eStatements, alerts, Interac e-Transfer messages and one-time
              verification codes. Choose an email address provided by a
              corporation or educational institution; for account security, ZippyX
              doesn&apos;t send one-time verification codes to personal or free
              email services.
            </p>
            <p>All fields are mandatory unless indicated (optional).</p>
          </Col>
          <Col md={4}>
            <Card className="p-3 bg-white">
              <Card.Title>You can also:</Card.Title>
              <Card.Body>
                <a
                  href="/"
                  className="text-decoration-none text-black"
                  style={{ fontSize: 12, marginTop: 5, marginBottom: 5 }}
                >
                  Change your address
                  <span style={{ color: 'rgb(220, 53, 69)', marginLeft: 5 }}>
                    {'>'}
                  </span>
                </a>
              </Card.Body>
            </Card>
          </Col>
        </Row>
        <Row>
          <Col md={12}>
            <div
              style={{
                borderBottom: '2px solid grey',
                paddingTop: 10,
                paddingBottom: 10,
                fontWeight: 'bold',
              }}
            >
              Contact Information
            </div>
            <Row className="pt-3 pb-3" style={{ borderBottom: '1px solid grey' }}>
              <Col md={3}>
                <span>Email address:</span>
              </Col>
              <Col md={9}>
                <Row>
                  <Col md={6}>
                    <Form.Control type="email" placeholder="Enter email" />
                  </Col>
                  <Col md={6}>
                    <Button variant="outline-danger">Send a test email</Button>
                  </Col>
                </Row>
                <p style={{ fontSize: 14, marginTop: 10 }}>
                  ZippyX will use this new email address to communicate with you for
                  many reasons including sending you promotional materials,
                  eBills, alerts and Interac e-Transfer messages.
                </p>
              </Col>
            </Row>
            <Row className="pt-3 pb-3" style={{ borderBottom: '1px solid grey' }}>
              <Col md={3}>
                <span>Email format:</span>
              </Col>
              <Col md={9}>
                <Form.Check
                  type="radio"
                  id="html"
                  label="HTML (includes graphics)"
                />
                <Form.Check type="radio" id="plain_text" label="Plain Text" />
              </Col>
            </Row>
            <Row className="pt-3 pb-3" style={{ borderBottom: '1px solid grey' }}>
              <Col md={3}>
                <span>Mobile phone:</span>
              </Col>
              <Col md={9}>
                <Row>
                  <Col md={6}>
                    <Form.Control type="text" placeholder="Enter phone number" />
                  </Col>
                </Row>
                <p style={{ fontSize: 14, marginTop: 10 }}>
                  If you choose to receive alerts by SMS (Short Message Service),
                  standard messaging and data rates from your carrier may apply.
                  We will send you a monthly reminder through text message every
                  30 days to allow you to opt out of receiving text messages from
                  us. This service is only available to Canadian wireless device
                  numbers.
                </p>
              </Col>
            </Row>
            <Row className="pt-3 pb-3" style={{ borderBottom: '1px solid grey' }}>
              <Col md={3}>
                <span>Home phone:</span>
                <div>(optional)</div>
              </Col>
              <Col md={9}>
                <Row>
                  <Col md={6}>
                    <Form.Control type="text" placeholder="Enter phone number" />
                  </Col>
                </Row>
              </Col>
            </Row>
            <Row className="pt-3 pb-3" style={{ borderBottom: '1px solid grey' }}>
              <Col md={3}>
                <span>Business phone:</span>
                <div>(optional)</div>
              </Col>
              <Col md={9}>
                <Row className="justify-content-center">
                  <Col md={3}>
                    <Form.Control type="text" placeholder="" />
                  </Col>
                  <Col md="auto" className="mt-2">
                    <span>Extension</span>
                  </Col>
                  <Col md={6}>
                    <Form.Control type="text" placeholder="" />
                    <p>
                      Phone numbers with extensions cannot be used to receive
                      alerts.
                    </p>
                  </Col>
                </Row>
              </Col>
            </Row>
            <Row className="mt-4 pb-4" style={{ borderBottom: '1px solid grey' }}>
              <Col>
                <Button variant="light" className="d-flex">
                  <div
                    style={{
                      width: 20,
                      height: 25,
                      border: '1px dotted grey',
                      textAlign: 'center',
                      marginRight: 10,
                    }}
                  >
                    P
                  </div>
                  Cancel
                </Button>
              </Col>
              <Col className="d-flex justify-content-end">
                <Button className="zippy-btn d-flex">
                  Save
                </Button>
              </Col>
            </Row>
            <div className="mt-2 pt-2">
              <div className="d-flex">
                <div
                  style={{
                    width: 20,
                    height: 25,
                    border: '1px dotted grey',
                    textAlign: 'center',
                    marginRight: 10,
                  }}
                >
                  P
                </div>
                <span style={{ fontWeight: 'bold' }}>Note:</span>
              </div>
              <ul>
                <li>
                  If you have chosen to receive alerts by SMS and you delete your
                  SMS mobile phone number, you will no longer receive alerts
                  through your mobile phone number
                </li>
              </ul>
            </div>
          </Col>
        </Row>
      </CommonPageContainer>
    </div>
  );
}
