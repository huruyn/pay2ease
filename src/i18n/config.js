import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

i18n.use(initReactI18next).init({
  fallbackLng: 'en-US',
  lng: 'en-US',
  resources: {
    'en-CA': {
      translations: require('./locales/en-CA/translations.json')
    },
    'en-US': {
      translations: require('./locales/en-US/translations.json')
    },
    'es-US': {
      translations: require('./locales/es-US/translations.json')
    },
    'fr-CA': {
      translations: require('./locales/fr-CA/translations.json')
    },
  },
  ns: ['translations'],
  defaultNS: 'translations'
});

i18n.languages = ['en-CA', 'en-US', 'es-US', 'fr-CA'];

export default i18n;