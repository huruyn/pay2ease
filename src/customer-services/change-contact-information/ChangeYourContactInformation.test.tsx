import '../../i18n/config';
import ChangeYourContactInformation from './ChangeYourContactInformation';

const ReactTestRenderer = require('react-test-renderer');

describe('ChangeYourContactInformation Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(<ChangeYourContactInformation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
