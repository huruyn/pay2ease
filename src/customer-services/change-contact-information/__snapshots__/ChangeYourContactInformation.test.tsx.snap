// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangeYourContactInformation Component matches the snapshot 1`] = `
<div
  className="page-content"
>
  <div
    className="d-flex justify-content-between align-items-center"
  >
    <h3>
      Change Your Contact Information
    </h3>
  </div>
  <div
    className="content-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8"
      >
        <div
          className="mt-3 row"
        >
          <div
            className="col"
          >
            <b>
              Step: 1 - 2
            </b>
          </div>
        </div>
        <h4
          className="mt-4"
        >
          Enter and verify your contact information
        </h4>
        <p
          className="mt-3"
        >
          ZippyX will use this new email address to communicate with you for many reasons including sending you promotional materials, eStatements, alerts, Interac e-Transfer messages and one-time verification codes. Choose an email address provided by a corporation or educational institution; for account security, ZippyX doesn't send one-time verification codes to personal or free email services.
        </p>
        <p>
          All fields are mandatory unless indicated (optional).
        </p>
      </div>
      <div
        className="col-md-4"
      >
        <div
          className="p-3 bg-white card"
        >
          <div
            className="card-title h5"
          >
            You can also:
          </div>
          <div
            className="card-body"
          >
            <a
              className="text-decoration-none text-black"
              href="/"
              style={
                Object {
                  "fontSize": 12,
                  "marginBottom": 5,
                  "marginTop": 5,
                }
              }
            >
              Change your address
              <span
                style={
                  Object {
                    "color": "rgb(220, 53, 69)",
                    "marginLeft": 5,
                  }
                }
              >
                &gt;
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          style={
            Object {
              "borderBottom": "2px solid grey",
              "fontWeight": "bold",
              "paddingBottom": 10,
              "paddingTop": 10,
            }
          }
        >
          Contact Information
        </div>
        <div
          className="pt-3 pb-3 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col-md-3"
          >
            <span>
              Email address:
            </span>
          </div>
          <div
            className="col-md-9"
          >
            <div
              className="row"
            >
              <div
                className="col-md-6"
              >
                <input
                  className="form-control"
                  placeholder="Enter email"
                  type="email"
                />
              </div>
              <div
                className="col-md-6"
              >
                <button
                  className="btn btn-outline-danger"
                  disabled={false}
                  type="button"
                >
                  Send a test email
                </button>
              </div>
            </div>
            <p
              style={
                Object {
                  "fontSize": 14,
                  "marginTop": 10,
                }
              }
            >
              ZippyX will use this new email address to communicate with you for many reasons including sending you promotional materials, eBills, alerts and Interac e-Transfer messages.
            </p>
          </div>
        </div>
        <div
          className="pt-3 pb-3 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col-md-3"
          >
            <span>
              Email format:
            </span>
          </div>
          <div
            className="col-md-9"
          >
            <div
              className="form-check"
            >
              <input
                className="form-check-input"
                disabled={false}
                id="html"
                type="radio"
              />
              <label
                className="form-check-label"
                htmlFor="html"
                title=""
              >
                HTML (includes graphics)
              </label>
            </div>
            <div
              className="form-check"
            >
              <input
                className="form-check-input"
                disabled={false}
                id="plain_text"
                type="radio"
              />
              <label
                className="form-check-label"
                htmlFor="plain_text"
                title=""
              >
                Plain Text
              </label>
            </div>
          </div>
        </div>
        <div
          className="pt-3 pb-3 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col-md-3"
          >
            <span>
              Mobile phone:
            </span>
          </div>
          <div
            className="col-md-9"
          >
            <div
              className="row"
            >
              <div
                className="col-md-6"
              >
                <input
                  className="form-control"
                  placeholder="Enter phone number"
                  type="text"
                />
              </div>
            </div>
            <p
              style={
                Object {
                  "fontSize": 14,
                  "marginTop": 10,
                }
              }
            >
              If you choose to receive alerts by SMS (Short Message Service), standard messaging and data rates from your carrier may apply. We will send you a monthly reminder through text message every 30 days to allow you to opt out of receiving text messages from us. This service is only available to Canadian wireless device numbers.
            </p>
          </div>
        </div>
        <div
          className="pt-3 pb-3 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col-md-3"
          >
            <span>
              Home phone:
            </span>
            <div>
              (optional)
            </div>
          </div>
          <div
            className="col-md-9"
          >
            <div
              className="row"
            >
              <div
                className="col-md-6"
              >
                <input
                  className="form-control"
                  placeholder="Enter phone number"
                  type="text"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="pt-3 pb-3 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col-md-3"
          >
            <span>
              Business phone:
            </span>
            <div>
              (optional)
            </div>
          </div>
          <div
            className="col-md-9"
          >
            <div
              className="justify-content-center row"
            >
              <div
                className="col-md-3"
              >
                <input
                  className="form-control"
                  placeholder=""
                  type="text"
                />
              </div>
              <div
                className="mt-2 col-md-auto"
              >
                <span>
                  Extension
                </span>
              </div>
              <div
                className="col-md-6"
              >
                <input
                  className="form-control"
                  placeholder=""
                  type="text"
                />
                <p>
                  Phone numbers with extensions cannot be used to receive alerts.
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="mt-4 pb-4 row"
          style={
            Object {
              "borderBottom": "1px solid grey",
            }
          }
        >
          <div
            className="col"
          >
            <button
              className="d-flex btn btn-light"
              disabled={false}
              type="button"
            >
              <div
                style={
                  Object {
                    "border": "1px dotted grey",
                    "height": 25,
                    "marginRight": 10,
                    "textAlign": "center",
                    "width": 20,
                  }
                }
              >
                P
              </div>
              Cancel
            </button>
          </div>
          <div
            className="d-flex justify-content-end col"
          >
            <button
              className="zippy-btn d-flex btn btn-primary"
              disabled={false}
              type="button"
            >
              Save
            </button>
          </div>
        </div>
        <div
          className="mt-2 pt-2"
        >
          <div
            className="d-flex"
          >
            <div
              style={
                Object {
                  "border": "1px dotted grey",
                  "height": 25,
                  "marginRight": 10,
                  "textAlign": "center",
                  "width": 20,
                }
              }
            >
              P
            </div>
            <span
              style={
                Object {
                  "fontWeight": "bold",
                }
              }
            >
              Note:
            </span>
          </div>
          <ul>
            <li>
              If you have chosen to receive alerts by SMS and you delete your SMS mobile phone number, you will no longer receive alerts through your mobile phone number
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="footer-content-wrapper"
  >
    <div
      className="quick-links container"
    >
      <div
        className="row"
      >
        <h3
          className="col-sm-12"
        >
          Quick Links
        </h3>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-2"
        >
          <ul>
            <li>
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li>
              Please Review Our Legal Stuff by visiting our
               
              <a
                className="text-decoration-none"
                href="/legal"
                onClick={[Function]}
              >
                Privacy Policy | TOU | EULA | CardHolder T&C Pages
              </a>
            </li>
          </ul>
        </div>
        <div
          className="col-sm-2"
        >
          8465 W Sahara Ave. Las Vegas, NEVADA
          <ul>
            <li>
              <a
                href="tel:+17026231876"
              >
                1.702.623.1876
              </a>
            </li>
            <li>
              <a
                href="mailto:contact@zippy.cash"
              >
                contact@zippy.cash
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 col-sm-12 col-xs-12 text-left"
        >
          © 2021 Zippy Cash LLC  All rights reserved.
          <br />
          <br />
          The Program Manager for the Zippy Cash® Prepaid Visa Debit Card is PayQuicker, LLC
          <br />
          Zippy Cash® is a registered trademark of Zippy Cash, LLC
          <br />
          The Zippy Cash Prepaid Visa Debit Card is issued by Metropolitan Commercial Bank (Member FDIC) pursuant to a license from Visa U.S.A. Inc. “Metropolitan Commercial Bank” and “Metropolitan” are registered trademarks of Metropolitan Commercial Bank ©2014. BY USING THIS CARD YOU AGREE TO THE TERMS AND CONDITIONS OF THE CARDHOLDER AGREEMENT AND FEE SCHEDULE, IF ANY. All trademarks and service marks are the property of their respective owners.
          <br />
          In compliance with the USA PATRIOT Act* and other laws, anyone who is a US resident over 18 with a valid Social Security number and a verifiable physical US address (no PO boxes or APO addresses) can apply for an Zippy Cash Card. This information must be provided, along with other information pertinent to receiving approval, when applying for a Zippy Cash Card. Currently, we do not issue the Zippy Cash Card to residents of Puerto Rico, Guam, the Virgin Islands, American Samoa, or other US territories.
          <br />
          *Important information for opening a card account: To help the federal government fight the funding of terrorism and money laundering activities, the USA PATRIOT Act requires all financial institutions and their third parties to obtain, verify and record information that identifies each person who opens a card account. What this means for you: When you open a card account, we will ask for your name, address, date of birth, social security number or country identification number, and other information that will allows us to identify you. We may also request a copy of your driver’s license or other documentation bearing your photo as verification of your identity.
          <br />
          Apple and the Apple logo are trademarks of Apple Inc., registered in the U.S. and other countries. App Store is a service mark of Apple Inc. Google Play is a trademark of Google Inc. Data rates may apply. Samsung Pay is a trademark of Samsung Inc.
        </div>
      </div>
    </div>
    <footer
      className="container mb-3"
    >
      <div
        className="d-none d-md-block"
      >
        <div
          className="d-flex justify-content-between border-bottom"
        />
        <div
          className="d-flex justify-content-between border-top"
        >
          <div
            className=""
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
          <div
            className="copyright mt-1"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
      <div
        className="d-block d-md-none"
      >
        <div
          className="d-flex border-bottom"
        >
          <div
            className="d-flex flex-column me-2"
          >
            <div
              className="footer-link mt-1"
            >
              <a
                className="text-decoration-none"
                href="/personal-profile"
              >
                dev docs
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Lore
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Ipsum
              </a>
            </div>
            <div
              className="footer-link mb-1"
            >
              <a
                className="text-decoration-none"
                href="/"
                onClick={[Function]}
              >
                Nullam Atoidio
              </a>
            </div>
          </div>
        </div>
        <div
          className="d-flex justify-content-end border-top"
        >
          <div
            className="copyright mt-2 text-end small"
          >
            <span>
              © 2021 Lorem ipsum dolor sit amet, consectetuer
            </span>
          </div>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
